Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# ci-fli-runner
+#
+
+# Apline 3.1 image.
+# https://hub.docker.com/_/alpine
```

Comment:
There's a minor misspelling in the comment here—`Apline`. I'd suggest removing the comment however, since the `FROM` instruction should be self explanatory and include both the name and version of the image. This means we can also avoid any dissociation between the comment and the instruction in future.

Reply:
Removed.